src/main/java/net/minecraftforge/fml/client/GuiWrongMinecraft.java

```diff
@@ -20,35 +20,32 @@
 package net.minecraftforge.fml.client;
 
 import net.minecraft.client.gui.GuiErrorScreen;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.util.text.TextFormatting;
 import net.minecraftforge.fml.common.Loader;
 import net.minecraftforge.fml.common.WrongMinecraftVersionException;
 
-public class GuiWrongMinecraft extends GuiErrorScreen
+public class GuiWrongMinecraft extends GuiErrorBase
 {
     private WrongMinecraftVersionException wrongMC;
     public GuiWrongMinecraft(WrongMinecraftVersionException wrongMC)
     {
-        super(null,null);
+        super();
```

Comment:
There is no reason to call `super()` without arguments, it is already called automatically when no other `super` call is made.
Please remove all calls to `super()`.